rebasebot/bot.py

```diff
@@ -579,8 +595,8 @@ def run(
         logging.info("Dry run mode is enabled. Do not create a PR.")
         return True
 
-    push_required = _is_push_required(gitwd, dest, source, rebase)
-    pr_url, pr_available = _is_pr_available(dest_repo, rebase)
+    push_required = _is_push_required(gitwd, dest, source, rebase, prs_with_manual_rebase_label)
+    pr_url, pr_available = _is_pr_available(dest_repo, rebase, prs_with_manual_rebase_label)
```

Comment:
What message is sent to slack when a PR is in manual mode? Already contains latest changes? Maybe we should have a specific this PR is in manual mode message posted, WDYT?

Reply:
```suggestion
    if not pr_available:
                # Case 4: in a list of PRs, there is atleast one with the rebase/manual label
                # that prevents the rebasebot from rebasing.
                _message_slack(slack_webhook, f"There is a PR in a manual mode.")
            else:
                # Case 5: source and dest repos are the same (git diff is empty), and there is no PR.
                # Just inform that there is nothing to update in the dest repository.
                _message_slack(
                    slack_webhook,
                    f"Destination repo {dest.url} already contains all latest changes."
                )
```

Reply:
I'm not sure that diff works quite, but yes, I'm looking for, PR in manual mode as some sort of log from it :)